main.cpp

```diff
@@ -109,7 +109,19 @@ Thing 4) Gas Grill
     3) turn of the flame
  */
 
-
+/*
+Thing 10) Wireless Headphone
+5 properties:
+    1) Speaker
+    2) Over Ear
```

Comment:
These are objects, not primitives!   You need to pick things that can be represented with `int float double bool char std::string`
- [ ] revise

Reply:
Ok I unterstand. I meant this here: https://images-na.ssl-images-amazon.com/images/I/51AoAGTUs6L._AC_.jpg
What's the correct english word for this?